widget/views/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8" />
+
+    <link rel="stylesheet" type="text/css" href="/static/index.css">
+    <title>브랜드페이 위젯 샘플 프로젝트</title>
+
+    <!-- BrandPay SDK script 추가 -->
+    <script src="https://js.tosspayments.com/v1/brandpay"></script>
+  </head>
+
+  <body>
+    <form id="payment-form">
+      <div class="wrapper vertical">
+        <div id="payment-methods-widget"></div>
+        <button id="submit" type="primary" class="button">결제하기</button>
+      </div>
+    </form>
+    <script>
+      const clientKey = 'test_ck_D5GePWvyJnrK0W0k6q8gLzN97Eoq';
+      const customerKey = 'customer123';
+
+      // brandpay 초기화
+      const brandpay = BrandPay(clientKey, customerKey, {
+        redirectUrl: window.location.origin + '/callback-auth',
```

Comment:
```suggestion
        // [TODO] 상점의 등록한 `redirectUrl`을 추가하세요.
        // 문서: https://docs.tosspayments.com/reference/brandpay-sdk#초기화-파라미터
        redirectUrl: window.location.origin + '/callback-auth',
```

의견) `redirectUrl` 관련 주석도 추가해주면 좋을것 같습니다 🙇

Reply:
다른 방식으로 추가해두었습니다~